nsxt/resource_nsxt_vpc_dhcp_v4_static_binding_config.go

```diff
@@ -0,0 +1,387 @@
+	"display_name": metadata.GetExtendedSchema(getDisplayNameSchema()),
+	"description":  metadata.GetExtendedSchema(getDescriptionSchema()),
+	"revision":     metadata.GetExtendedSchema(getRevisionSchema()),
+	"tag":          metadata.GetExtendedSchema(getTagsSchema()),
+	"parent_path":  metadata.GetExtendedSchema(getPolicyPathSchema(true, true, "Policy path of the parent")),
+	"gateway_address": {
+		Schema: schema.Schema{
+			Type:         schema.TypeString,
+			ValidateFunc: validation.IsIPv4Address,
+			Optional:     true,
+		},
+		Metadata: metadata.Metadata{
+			SchemaType:   "string",
+			SdkFieldName: "GatewayAddress",
+			OmitIfEmpty:  true,
+		},
+	},
+	"host_name": {
+		Schema: schema.Schema{
+			Type:     schema.TypeString,
+			Optional: true,
+		},
+		Metadata: metadata.Metadata{
+			SchemaType:   "string",
+			SdkFieldName: "HostName",
+		},
+	},
+	"mac_address": {
+		Schema: schema.Schema{
+			Type:         schema.TypeString,
+			ValidateFunc: validation.IsMACAddress,
+			Optional:     true,
+		},
+		Metadata: metadata.Metadata{
+			SchemaType:   "string",
+			SdkFieldName: "MacAddress",
+		},
+	},
+	"lease_time": {
+		Schema: schema.Schema{
+			Type:     schema.TypeInt,
+			Optional: true,
+			Default:  86400,
+		},
+		Metadata: metadata.Metadata{
+			SchemaType:   "int",
+			SdkFieldName: "LeaseTime",
+		},
+	},
+	"ip_address": {
+		Schema: schema.Schema{
+			Type:         schema.TypeString,
+			ValidateFunc: validation.IsIPv4Address,
+			Optional:     true,
+		},
+		Metadata: metadata.Metadata{
+			SchemaType:   "string",
+			SdkFieldName: "IpAddress",
+		},
+	},
+	"options": {
+		Schema: schema.Schema{
+			Type:     schema.TypeList,
+			MaxItems: 1,
+			Elem: &metadata.ExtendedResource{
+				Schema: map[string]*metadata.ExtendedSchema{
+					"option121": {
+						Schema: schema.Schema{
+							Type:     schema.TypeList,
+							MaxItems: 1,
+							Elem: &metadata.ExtendedResource{
+								Schema: map[string]*metadata.ExtendedSchema{
+									"static_route": {
+										Schema: schema.Schema{
+											Type: schema.TypeList,
+											Elem: &metadata.ExtendedResource{
+												Schema: map[string]*metadata.ExtendedSchema{
+													"next_hop": {
+														Schema: schema.Schema{
+															Type:         schema.TypeString,
+															ValidateFunc: validateSingleIP(),
+															Optional:     true,
+														},
+														Metadata: metadata.Metadata{
+															SchemaType:   "string",
+															SdkFieldName: "NextHop",
+														},
+													},
+													"network": {
+														Schema: schema.Schema{
+															Type:         schema.TypeString,
+															ValidateFunc: validateCidrOrIPOrRange(),
+															Optional:     true,
+														},
+														Metadata: metadata.Metadata{
+															SchemaType:   "string",
+															SdkFieldName: "Network",
+														},
+													},
+												},
+											},
```

Comment:
We'll avoid edge case issues by making this required (no other reason to specify `option121`)